### ISSUE TITLE
Provide presto-admin on docker image

### ISSUE THREAD

**Issue by kokosing** [prestodb/presto-admin#152]
Each build should push docker image with presto-admin installed into docker-hub under `teradata-labs` organization so user can easily install Presto on docker cluster.

This could be done automatically by travis. 

Image should be tagged with current version. Snapshot and release version should be tagged differently.

**Comment by kokosing** [comment]
See #143 .

**Comment by kokosing** [comment]
#143

**Comment by petroav** [comment]
I don't think we should push out a new docker image for every build. It makes more sense to push something for every release.
